client/src/components/DataFiles/DataFilesProjectMembers/_cells/SystemRoleSelector.jsx

```diff
@@ -0,0 +1,100 @@
+import React, { useState, useEffect, createContext, useContext } from 'react';
+import { useQuery, useMutation } from 'react-query';
+import { useSelector } from 'react-redux';
+import Cookies from 'js-cookie';
+import fetch from 'cross-fetch';
+import DropdownSelector from '_common/DropdownSelector';
+import { Button } from 'reactstrap';
+import styles from '../DataFilesProjectMembers.module.scss';
+import LoadingSpinner from '_common/LoadingSpinner';
+
+const getSystemRole = async (projectId, username) => {
+  const url = `/api/projects/${projectId}/system-role/${username}/`;
```

Comment:
Getting endless runaway queries to `/api/projects/LOCAL.CEP-101/system-role/${username}/` for users when opening a project "Manage Team" modal

Reply:
Good catch, I was using a query result in `columnMemoProps` and it was causing an infinite render loop because the js equality operator is cursed. Fixed by changing the query object to the primitive `role` string.